frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-uninstall.cy.ts

```diff
@@ -93,11 +88,15 @@ xdescribe(`Testing uninstall of ${testOperator.name} Operator`, () => {
     modal.shouldBeClosed();
   });
 
-  it(`attempts to uninstall the Operator, shows 'Error uninstalling Operator' alert`, () => {
+  xit(`attempts to uninstall the Operator, shows 'Error uninstalling Operator' alert`, () => {
```

Comment:
Is this test broken?

Reply:
Will fix as a follow on in https://github.com/openshift/console/pull/13214 (which will require a couple tweaks).  The PR was authored to work with the existing version of Cypress but didn't actually work because part of the fix comes with the Cypress upgrade.